src/index.html

```diff
@@ -9,6 +12,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-link">
```

Comment:
Please wrap `<a/>` with `<div />` - div's are more predictable when it comes to `flex-box`, and moving things around the page generally

Reply:
done in solution2